README.rst

```diff
@@ -3,9 +3,101 @@ Open edX Devstack |Build Status|
 
 Tahoe Devstack Docs
 ===================
-To run a Tahoe devstack, follow the same instructions in the "Prerequisites" section,
-but use ``$ make tahoe.up`` instead of ``$ make dev.up`` to use the Tahoe specific
-features and environment settings.
+To run a Tahoe devstack follow the steps below:
+
+This project requires **Docker 17.06+ CE**.
+
+.. code::
+
+    $ make dev.clone
+    $ make dev.provision
```

Comment:
I cloned the devstack appsembler fork. Did not change branch (so branch is `hawthorn`)
After cloning the appsembler devstack fork and cd'ing to that directory, I ran just the commands specified above

`make dev.clone` This seems to have worked fine

`make dev.provision` this seems to have worked well until here:

```
---> pavelib.assets.webpack
python manage.py lms --settings=devstack_docker print_setting STATIC_ROOT 2>/dev/null
python manage.py cms --settings=devstack_docker print_setting STATIC_ROOT 2>/dev/null
python manage.py lms --settings=devstack_docker print_setting WEBPACK_CONFIG_PATH 2>/dev/null
NODE_ENV=development STATIC_ROOT_LMS=DISABLING CACHES...
/edx/var/edxapp/staticfiles STATIC_ROOT_CMS=DISABLING CACHES...
/edx/var/edxapp/staticfiles/studio $(npm bin)/webpack --config=DISABLING CACHES...
webpack.dev.config.js
/bin/sh: 1: CACHES...: not found
/bin/sh: 2: /edx/var/edxapp/staticfiles: Permission denied
/bin/sh: 3: /edx/var/edxapp/staticfiles/studio: Permission denied
/bin/sh: 4: webpack.dev.config.js: not found

...

Build failed running pavelib.assets.update_assets: Subprocess return code: 127
make: *** [dev.provision.run] Error 1
```

Reply:
I started over with a clean install and get the same error

Reply:
@johnbaldwin confirming that I get the same error on a clean `make dev.provision`.

Reply:
@johnbaldwin @thraxil thanks for the report. I've found and fixed the error which used to break the step `$ make lms-static`.

Please checkout this pull request https://github.com/appsembler/edx-platform/pull/375 and let me know.